.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,12 +5,12 @@ Thank you for taking the time to contribute to The Odin Project. In order to get
  - [ ] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md).
  - [ ] The title summarizes the change and where it happened, for example: "Fixes punctuation in Clean Code lesson".
  - [ ] If the PR is related to an open issue, use a [relevant keyword](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests) and reference it with the `#` sign and the issue number.
-- [ ] If changes are requested, please make the changes in a timely manner. After you submit the changes, request another review from the maintainer (top right).
+ - [ ] If changes are requested, please make the changes in a timely manner. After you submit the changes, request another review from the maintainer (top right).
 
-#### 1.Describe the changes made and include why they are necessary or important:
+#### 1. Describe the changes made and include why they are necessary or important:
 
 ...your text here
 
 #### 2. Related Issue
 
-keyword #XXXXX
+Closes #XXXXX
```

Comment:
Can we safely use the "Closes keyword" here with our current issue structure in some cases? Think: Knowledge checks for a given section.

Would linking to the keywords in the template create unnecessary overhead in this section?

Reply:
IMO the knowledge check issues are actually a perfect example of why this would be a beneficial change.

The massive multi-PR issues that were originally used there were far from easily maintainable for us or easily accessible to the community, which were the main drivers for breaking them all out into individual issues.

I think that the vast majority of our issues should be (and usually are, especially on this repo) single PR closable, for the same reasons that the original knowledge check issues were closed and replaced.